### PR TITLE
Update docs for Elixir v6

### PR DESCRIPTION
**Pull request by JeffreyWay**
Yo, here are some doc updates for Elixir 6.

image

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
:rocket:

File: elixir.md [hunk 1]
```diff
@@ -145,6 +150,29 @@ elixir(function(mix) {
...
 
+While it's recommended that you use Laravel's default asset directories, if you require a different base directory, you may begin any file path with `./`. This instructs Elixir to begin at the project root, rather than using the default base directory.
+
+```javascript
+elixir(function(mix) {
+    mix.sass('./app/assets/app.scss');
+});
+```
+
+With this adjustment, Elixir will now compile `./app/assets/sass/app.scss` to `public/css/app.css`.
```
Comment (michaeldyrynda) [inline elixir.md]: Should this read `./app/assets/app.scss`? Or should line 157 read `./app/assets/sass/app.scss`?

File: elixir.md [hunk 2]
```diff
@@ -159,23 +187,23 @@ elixir(function(mix) {
...
-    ], 'public/assets/css');
+    ], 'public/assets/css/site.css');
 });
 ```
 
 <a name="css-source-maps"></a>
 ### Source Maps
 
-Source maps are enabled out of the box. So, for each file that is compiled you will find a companion `*.css.map` file in the same directory. This mapping allows you to trace your compiled stylesheet selectors back to your original Sass or Less while debugging in your browser.
+Source maps are enabled out of the box. For each relevant file that is compiled, you will find a companion `*.css.map` or `*.js.map` file in the same directory. This mapping allows you to trace your compiled CSS selectors or scripts back to your original source file while debugging in your browser.
```
Comment (michaeldyrynda) [inline elixir.md]: Is this both with and without the `--production` switch?

File: elixir.md [hunk 3]
```diff
@@ -219,56 +238,83 @@ elixir(function(mix) {
...
-        'react-component.jsx'
-    ]);
+    mix.webpack('app.js');
+});
+```
+
+
+To choose a different output location, simply specify your desired path as the second argument.
+
+If you'd like to leverage more of Webpack's functionality, this task will read any `webpack.config.js` file in your project root, and [factor its configuration](https://webpack.github.io/docs/configuration.html) into the build process. Alternatively, you may pass your Webpack-specific configuration as the fourth argument to `mix.webpack()`.
```
Comment (michaeldyrynda) [inline elixir.md]: s/fourth/third [argument]? Or should the third argument be documented as well?

File: elixir.md [hunk 4]
```diff
@@ -395,13 +444,12 @@ If you need more flexibility than Elixir's `task` method can provide, you may cr
 
 var gulp = require('gulp');
 var shell = require('gulp-shell');
-var Elixir = require('laravel-elixir');
 
 var Task = Elixir.Task;
```
Comment (arthurkirkosa) [inline elixir.md]: Should this variable be removed?

Comment (JeffreyWay) [inline elixir.md]: Depends on where you create the extension.

File: elixir.md (hunk 1 above)
Comment (JeffreyWay) [inline elixir.md]: Yep - thanks!

File: elixir.md (hunk 3 above)
Comment (JeffreyWay) [inline elixir.md]: I'll add a note about the third arg.

**Comment by taylorotwell** [comment]
This was done.

**Pull request closed without merging**